lib/cron/cache-config.ts

```diff
@@ -1,6 +1,9 @@
 import { Protocol } from '@uniswap/router-sdk'
 import { V2SubgraphProvider, V3SubgraphProvider, V4SubgraphProvider } from '@uniswap/smart-order-router'
 import { ChainId } from '@uniswap/sdk-core'
+import dotenv from 'dotenv'
+
+dotenv.config()
```

Comment:
is this for local testing?

Reply:
yep, thought it might fix the alchemy subgraph API key loading issue from the local .env file.

Reply:
I just confirmed on my local routing-api instance, that it does.